packages/server/src/koaJsonRpc/lib/methodConfiguration.ts

```diff
@@ -171,4 +171,7 @@ export const methodConfiguration = {
   eth_getFilterLogs: {
     total: tier2rateLimit,
   },
+  debug_traceTransaction: {
+    total: tier2rateLimit,
```

Comment:
Probably this should be tier1 as it's more heavier task making several mirror-node requests.